components/homepage/work/WorkCard.tsx

```diff
@@ -2,28 +2,28 @@ import Image from 'next/image';
 import Link from 'next/link';
 
 import { WorkCardType } from '@/types';
-import MotionWrapperCardDiv from '../../HOC/MotionWrapperCardDiv';
-import { LinkIcon } from '../icons/LinkIcon';
+import MotionWrapperCardDiv from '../../../HOC/MotionWrapperCardDiv';
+import { LinkIcon } from '../../icons/LinkIcon';
 
 const WorkCard = ({
   title,
   subtitle,
   description,
   imgSrc,
   index,
-  certified,
+
   certificateLink,
-  _id,
 }: WorkCardType) => {
   return (
     <MotionWrapperCardDiv
-      direction="right"
+      direction="up"
       type="spring"
       delay={0.05 * index}
       duration={1.75}
-      additionalStyles="flex flex-col items-start  rounded-md bg-white-800 transition-transform duration-300 hover:translate-x-[2rem] hover:bg-white-900 dark:bg-black-200 dark:hover:bg-black-300"
+      translateX="1.5rem"
+      additionalStyles="hover:dark:shadow-servicesCard dark:hover:shadow-black/50 hover:shadow-2xl pb-3 dark:hover:shadow-black-900 rounded-xl overflow-hidden"
     >
-      <div className="flex items-center justify-start">
+      <div className="flex  items-center justify-start">
```

Comment:
```suggestion
      <div className="flex items-center justify-start">
```

Reply:
You may want to check your eslint and prettier settings if this doesn't automatically get caught.